app/providers/atmosphere/optimization_strategy/default.rb

```diff
@@ -105,7 +120,7 @@ def select
           best_tenant = nil
           best_flavor = nil
           instantiation_cost = Float::INFINITY
-          tmpls.each do |tmpl|
+          @tmpls.each do |tmpl|
```

Comment:
E.g. here https://github.com/dice-cyfronet/atmosphere/blob/optimize_tenant_selection_2/app/providers/atmosphere/optimization_strategy/default.rb#L245 and in other places `tmpls` getter is used.